File: internal/enums/message.go
package enums

type MessageType string
type MessageColor string

const (
	MessageTypeSuccess MessageType = "success"
	MessageTypeError   MessageType = "error"
	MessageTypeInfo    MessageType = "info"
	MessageTypeConfig  MessageType = "config"
	MessageTypeWarning MessageType = "warning"
)

const (
	MessageColorGreen   MessageColor = "32"
	MessageColorBlue    MessageColor = "36"
	MessageColorRed     MessageColor = "31"
	MessageColorWhite   MessageColor = "37"
	MessageColorYellow  MessageColor = "33"
	MessageColorDefault MessageColor = "0"
)

File: src/enums/message.rs
#[derive(Hash, PartialEq, Eq)]
pub enum MessageType {
    Success,
    Error,
    Warning,
    Info,
    Config,
}

pub enum MessageColor {
    Green = 32,
    Blue = 36,
    Red = 31,
    White = 37,
    Yellow = 33,
    Default = 0,
}
